components/src/core/components/ListTable/CellContainer.vue

```diff
@@ -90,7 +90,7 @@ export default defineComponent({
           item: cellData,
           header: header,
           rowItem: rowData,
-          class: header.class,
+          class: header.name === 'selector' ? [header.class, 'checkbox-cell'] : header.class,
```

Comment:
when you add the class here it will go inside the cell. below you have already added it once outside. is this intentional ?

Reply:
I thought it was needed, but I checked it and found out its not required.